aiida/schedulers/plugins/slurm.py

```diff
@@ -214,6 +214,19 @@ def _get_joblist_command(self, jobs=None, user=None):
                 if not isinstance(jobs, (tuple, list)):
                     raise TypeError("If provided, the 'jobs' variable must be a string or a list of strings")
                 joblist = jobs
+
+            # Trick: If we are asking for a single job, we append the same job once more.
+            # We do this because squeue returns a non-zero exit code if provided an invalid jobid
+            # (stderr: "slurm_load_jobs error: Invalid job id specified"), which can happen if time has passed since
+            # the job finished and squeue no longer has knowledge of it.
+            # However, this *only* happens when providing a *single* jobid to squeue.
+            # When providing two or more jobids via `squeue --jobs=123,234`, it always returns exit code zero, and
+            # duplicating job ids has no other effect on the output.
```

Comment:
Perhaps clearer like this
```suggestion
            # However, this *only* applies when providing a *single* jobid to squeue.
            # When providing two or more jobids via `squeue --jobs=123,234`, squeue stops caring whether they are valid
            #  and returns exit code zero irrespectively (duplicating job ids has no other effect on the output).
```

Reply:
I haver provided a change in the suggestion above